platform/macos/os_macos.h

```diff
@@ -117,7 +117,8 @@ class OS_MacOS : public OS_Unix {
 	virtual Vector<String> get_granted_permissions() const override;
 	virtual void revoke_granted_permissions() override;
 
-	virtual bool _check_internal_feature_support(const String &p_feature) override;
+	virtual void _register_compiled_feature() override;
+	virtual bool OS_MacOS::_check_internal_feature_support(const String &p_feature) override;
```

Comment:
Implementation in the `os_macos.mm` seems to be missing.

Reply:
And OS_Web from the abstract errors

Reply:
Still needed for `OS_Web`